### DISCUSSION TITLE
PolyGlot should be Flatpak.

### DISCUSSION THREAD

**Discussion by theJetway** [DraqueT/PolyGlot#1380] [Ideas]
So people on non Debian-based distros could also use this app.

**Comment by synthfi** [comment]
this is in the plans. I need to figure out the best way to make a flatpak using the existing output of the `jpackage` tool used for generating installers. Just haven't had time to get back around to it, but I anticipate that I'll have progress on it next January

**Reply by theJetway** [reply]
nice
